independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/catalog/processor/CatalogProcessor.java

```diff
@@ -35,6 +42,9 @@ public static CatalogProcessor of(ExtensionCatalog catalog) {
                 orderedCategories.add(new ProcessedCategory(c, extsByCategory.get(c.getId())));
             }
         }
+        JsonCategory category = new JsonCategory();
+        category.setId(UNCATEGORIZED);
```

Comment:
Could you also set name: `Uncategorized` and description: `The category is not defined for those extensions.`

Reply:
Done!